yjit/src/backend/ir.rs

```diff
@@ -857,21 +868,34 @@ pub struct Assembler
     /// Index of the last insn using the output of this insn
     pub(super) live_ranges: Vec<usize>,
 
+    /// Parallel vec with insns
+    /// Bitmap of spilled stack slot indexes of this insn
+    pub(super) spilled_temps: Vec<SpilledTemps>,
```

Comment:
This is to store the spilled temps state at different points in the assembler?

Reply:
Yes. We need to calculate state transitions in `push_insn` for cross-Assembler state propagation, and I thought we could reuse the results in `alloc_temp_regs` if we store this. However, I ended up copying almost the same logic to `alloc_temp_regs` while safely migrating the code to use a bitmap, so it's currently not utilized that well though. (to be cleaned up after more work is done)